api_tests/lib_sdk/actors/actor.ts

```diff
@@ -241,6 +241,30 @@ export class Actor {
         }
     }
 
+    public async overfund() {
+        const response = await this.swap.tryExecuteAction("fund", {
+            maxTimeoutSecs: 10,
+            tryIntervalSecs: 1,
+        });
+        const amount = response.data.payload.amount;
+        response.data.payload.amount = amount * 1.01;
```

Comment:
Just to be on the save side, what do you think of changing this to this? 
I’m thinking of catching rounding errors in advance.

```suggestion
        response.data.payload.amount = amount + 100 ;
```

Reply:
nobody put the `no-mergify` label, so it merged :)
But happy to still reply to this (same for the other comment):
I think the multiplication is actually safer, because the amount depends on the currency and the supply. With the multiplication we always add a fraction  relative to the amount rather than a fixed amount.

Reply:
> With the multiplication we always add a fraction relative to the amount rather than a fixed amount.

That is exactly the problem. 
If you are swapping 10 wei/satoshi, this would fail or not have any impact. Which means, we just have to have high enough values.